content/blocksPages/about.json

```diff
@@ -100,31 +99,31 @@
       "items": [
         {
           "headline": "2024: Acquired by SSW",
-          "content": "SSW loved using Tina so much that they reached out to help and ended up acquiring TinaCMS, bringing greater enterprise resources, support, and expertise. \n\nVersion 2 was released, with big updates including improved content indexing and better performance. Additionally, the team launched the TinaGPT chatbot in June, integrating AI-driven assistance into the platform.\n"
+          "content": "SSW loved using Tina so much that they reached out to help and **[SSW ended up acquiring TinaCMS](https://adamcogan.com/2024/05/15/ssw-purchases-tinacms/)**, bringing greater enterprise resources, support, and expertise. \n\n**Version 2** was released, with big updates including improved content indexing and better performance. Additionally, the team launched the TinaGPT chatbot in June, integrating AI-driven assistance into the platform.\n"
         },
         {
           "headline": "2023: 10,000 stars on GitHub ⭐  ",
-          "content": "Tina reached some exciting milestones, with 1,000 members in our Discord support forum and 10,000 GitHub stargazers, thanks to the amazing Tina community 🥰 \n\nNew features were added, like block-level control, schema-based content models, and enhanced UI customization, allowing for more sophisticated content structuring. \n"
+          "content": "Tina reached some exciting milestones: **[now fully open-source](<https://tina.io/blog/Tinacms-is-now-fully-open-source/ >)**, Tina reached 1,000 members in our Discord support forum and 10,000 GitHub stargazers, thanks to the amazing Tina community 🥰 \n\nTinaCloud added enterprise features like [Single Sign On and an Editorial Workflow](https://tina.io/blog/Introducing-Editorial-Workflow-Features-for-your-Team).\n"
```

Comment:
why is the link for 'now fully open-source' wrapped in < > ?

Reply:
That is a great question I don't know the answer to. It can be fixed.
I did it the same way as the other links:
- Select text
- Select the link button 🔗
- Copy/Paste link
- Select update